From 60c60b8c57c8f48baf979833c03ffc8391112375 Mon Sep 17 00:00:00 2001
From: Lin Jianhua <linjh@rock-chips.com>
Date: Thu, 29 Nov 2018 15:17:48 +0800
Subject: [PATCH] ARM64: dts: rockchip: rk3308: add rk3308k dtsi use to support
 wide temperature chip

Ambient temperature range: -20~85
Supported chips: RK3308K/RK3308GK/RK3308BK/RK3308HK

Change-Id: I40f23e20b2323a5a7d93cd022382b5eda9b6a3a0
Signed-off-by: Lin Jianhua <linjh@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3308k.dtsi | 28 +++++++++++++++++++++++
 1 file changed, 28 insertions(+)
 create mode 100644 arch/arm64/boot/dts/rockchip/rk3308k.dtsi

diff --git a/arch/arm64/boot/dts/rockchip/rk3308k.dtsi b/arch/arm64/boot/dts/rockchip/rk3308k.dtsi
new file mode 100644
index 000000000000..fac8d1bb8ac6
--- /dev/null
+++ b/arch/arm64/boot/dts/rockchip/rk3308k.dtsi
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/*
+ * Copyright (c) 2017 Fuzhou Rockchip Electronics Co., Ltd
+ */
+
+#include <dt-bindings/input/input.h>
+#include "rk3308.dtsi"
+
+&cpu0_opp_table {
+	opp-408000000 {
+		/delete-property/ opp-suspend;
+	};
+	opp-1008000000 {
+		opp-suspend;
+	};
+};
+
+&thermal_zones {
+	soc-thermal {
+		sustainable-power = <198>;
+
+		trips {
+			trip-point-0 {
+				temperature = <55000>;
+			};
+		};
+	};
+};
-- 
2.35.3

